src/main/java/seedu/nuscents/ui/Ui.java

```diff
@@ -93,15 +94,19 @@ public static void showTransactionList(TransactionList transactionList) {
             }
             if (transaction instanceof Allowance) {
                 System.out.printf("%-5s  %-10s  %-7s  %-18s  %-15s  %-5s %n", index, "Allowance",
-                        "$" + transaction.getAmount(), transaction.getFormattedDate(), transaction.getDescription(),
+                        "$" + String.format("%.2f", transaction.getAmount()), transaction.getFormattedDate(), transaction.getDescription(),
                         note);
+                netBalance += transaction.getAmount();
             } else if (transaction instanceof Expense) {
                 System.out.printf("%-5s  %-10s  %-7s  %-18s  %-15s  %-5s %n", index, "Expense",
-                        "$" + transaction.getAmount(), transaction.getFormattedDate(), transaction.getDescription(),
+                        "$" + String.format("%.2f", transaction.getAmount()), transaction.getFormattedDate(), transaction.getDescription(),
```

Comment:
Code is longer than 120 characters which violates the coding standard, perhaps can use 2 lines for the code

Reply:
Ok edited, hopefully it works now!